journal/modernized/02-chapter-2.adoc

```diff
@@ -635,7 +637,7 @@ a state of resistance of the clay in the hands of the great Potter was treated w
 and the danger of standing out compared to rebellion, which is as the sin of witchcraft.
 
 16th.--Waterford meeting.
-After we had settled down into silence at 's in the evening,
+After we had settled down into silence at _______'s in the evening,
```

Comment:
This is great, we always want groups of underscores to indicated places where names have been removed. 👍

The only thing we need to tweak here is a weird thing caused by the nature of the asciidoc format.  When we do these groups of underscores, we have to wrap them with three plus signs, in order to keep asciidoc from getting confused, like this:

```
After we had settled down into silence at +++_______+++'s in the evening,
```

And you got the correct number of underscores, we always use 7.  Here's the spot of the styleguide that shows/explains some about the plus signs:

https://flp-styleguide.netlify.com/#gotchas (scroll down a bit)